templates/music/album_detail.html

```diff
@@ -4,215 +4,75 @@
 
 {% block extrajs %}
-    <script type="text/javascript" src="{% static "js/Chart.js" %}"></script>
-    <script type="text/javascript">
-        {% if estimated_cumulative_revenue %}
-            var lineChartData = {
-                labels: [
-                    {% for month_revenue in estimated_cumulative_revenue %}
-                        "{{ month_revenue.month }}"{% if not forloop.last %},{% endif %}
-                    {% endfor %}
-                ],
+    <script type="text/javascript" src="{% static "js/vendor/Chart.js" %}"></script>
+
+    <script>
+        var ctx = document.getElementById('myChart').getContext('2d');
+        var myChart = new Chart(ctx, {
+            type: 'line',
+            data: {
+                labels: [{% for month_revenue in estimated_cumulative_revenue %}"{{ month_revenue.month }}"{% if not forloop.last %},{% endif %}{% endfor %}],
                 datasets: [
                     {
-                        label: "Artist Revenue",
-                        fillColor: "rgba(220,220,220,0.2)",
-                        strokeColor: "rgba(220,220,220,1)",
-                        pointColor: "rgba(220,220,220,1)",
-                        pointStrokeColor: "#fff",
-                        pointHighlightFill: "#fff",
-                        pointHighlightStroke: "rgba(220,220,220,1)",
-                        data: [
-                            {% for month_revenue in estimated_cumulative_revenue %}
-                                {{ month_revenue.artist_revenue }}{% if not forloop.last %},{% endif %}
-                            {% endfor %}
-                        ]
+                        label: 'Downloads',
+                        data: [{% for disc in album.discs %}{% for track in disc %}{% if track.total_downloads %}{{ track.total_downloads }},{% endif %}{% endfor %}{% endfor %}],
+                        backgroundColor: "#34495E"
                     },
                     {
-                        label: "Investor Income",
-                        fillColor: "rgba(151,187,205,0.2)",
-                        strokeColor: "rgba(151,187,205,1)",
-                        pointColor: "rgba(151,187,205,1)",
-                        pointStrokeColor: "#fff",
-                        pointHighlightFill: "#fff",
-                        pointHighlightStroke: "rgba(151,187,205,1)",
-                        data: [
-                            {% for month_revenue in estimated_cumulative_revenue %}
-                                {{ month_revenue.investor_revenue }}{% if not forloop.last %},{% endif %}
-                            {% endfor %}
-                        ]
+                        label: 'Streams',
+                        data: [{% for disc in album.discs %}{% for track in disc %}{% if track.total_streams %}{{ track.total_streams}},{% endif %}{% endfor %}{% endfor %}],
+                        backgroundColor: "#85929E"
                     }
-                ]
+                ],
```

Comment:
Remove the trailing comma at the end of this line. This JS will break in some browsers.